cdap-master/src/main/java/io/cdap/cdap/master/environment/k8s/PreviewServiceMain.java

```diff
@@ -95,10 +101,12 @@ protected CConfiguration updateCConf(CConfiguration cConf) {
   @Override
   protected List<Module> getServiceModules(MasterEnvironment masterEnv,
                                            EnvironmentOptions options, CConfiguration cConf) {
-    List<Module> modules = new ArrayList<>(Arrays.asList(
+    List<Module> modules = new ArrayList<>();
+    modules.addAll(Arrays.asList(
       new DataSetServiceModules().getStandaloneModules(),
       new DataSetsModules().getStandaloneModules(),
-      new AppFabricServiceRuntimeModule().getStandaloneModules(),
+      Modules.override(new AppFabricServiceRuntimeModule(cConf).getStandaloneModules())
+        .with(new AuthenticationContextModules().getInternalAuthMasterModule(cConf)),
```

Comment:
yep,  I see similar override here as in twill one. Please move to AppFabricServiceRuntimeModule and reuse

Reply:
Fixed